backend/python-langchain/main.py

```diff
@@ -49,18 +55,11 @@
 api_url = getenv('LANGCHAIN_UPLOAD_ENDPOINT')
 delete_url = getenv('LANGCHAIN_DELETE_ENDPOINT')
 cors_domains = getenv('CORS_DOMAINS').split(',')
-cookie_domain = getenv('COOKIE_DOMAIN')
 app = Flask(__name__)
 app.secret_key = getenv('SECRET_KEY', '')
 app.config['SESSION_TYPE'] = 'filesystem'
-app.config['SESSION_COOKIE_SAMESITE'] = 'None'
```

Comment:
@felipegm0911 please return back all these cookie configurations, these are needed to handle the front end connection